templates/pages/admin/form/form_destination_commonitil_config.html.twig

```diff
@@ -31,223 +30,398 @@
+        {% if field.canHaveMultipleStrategies() %}
+            <template data-glpi-itildestination-field-config-template>
+                <section
+                    class="mb-2"
+                    data-glpi-itildestination-field-config
+                >
+                    {{ fields.dropdownArrayField(
+                        '%s[%s][]'|format(
+                            item.formatConfigInputName(field.getKey()),
+                            config_field.getStrategiesInputName()
+                        ),
+                        '',
+                        field.getStrategiesForDropdown(),
+                        '',
+                        {
+                            is_horizontal: false,
+                            field_class: '',
+                            no_label: true,
+                            mb: '',
+                            display_emptychoice: true,
+                            init: false,
+                            aria_label: __('Select strategy...'),
+                        }|merge(extra_options)
+                    ) }}
+                    {{ field.renderConfigForm(
+                        form,
+                        config_field,
+                        item.formatConfigInputName(field.getKey()),
+                        {
+                            'is_horizontal': false,
+                            'init': false,
+                        }|merge(extra_options)
+                    )|raw }}
+                    <button
+                        type="button"
+                        class="btn btn-outline"
+                        title="{{ __('Remove strategy') }}"
+                        aria-label="{{ __('Remove strategy') }}"
+                        data-glpi-itildestination-remove-field-config
+                    >
+                        <i class="ti ti-x"></i>
+                    </button>
                 </section>
+            </template>
+        {% endif %}
+    </section>
+
+    <script>
+        import("{{ js_path('js/modules/Forms/FieldDestinationMultipleConfig.js') }}").then((m) => {
+            new m.GlpiFormFieldDestinationMultipleConfig(
+                $('[data-glpi-itildestination-field="{{ field_container_rand }}"]'),
+            );
+        });
+    </script>
+{% endmacro %}
+
+{% set left_regular_cls = "col-lg-8" %}
+{% set right_regular_cls = "col-lg-4" %}
 
-                <script>
-                    import("{{ js_path('js/modules/Forms/FieldDestinationMultipleConfig.js') }}").then((m) => {
-                        new m.GlpiFormFieldDestinationMultipleConfig(
-                            $('[data-glpi-itildestination-field="{{ field_container_rand }}"]'),
-                        );
-                    });
-                </script>
+{% set left_expanded_cls = "col-xl-5 col-lg-6" %}
+{% set right_expanded_cls = "col-xl-7 col-lg-6" %}
+
+{% set left_side_cls = left_regular_cls %}
+{% set right_side_cls = right_regular_cls %}
+{% if is_expanded %}
```

Comment:
I think this was copied from the ticket UI where the panel can be extended.
If it can't be done here, you probably don't need all these variables.